installer.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+VERSION=$1
+OS=$(uname | awk '{print tolower($0)}')
+arch=$(uname -m)
+case ${arch} in
+  x86_64)
+    ARCH="amd64"
+    ;;
+  i386|i686)
+    ARCH="386"
+    ;;
+  armv8l|aarch64)
+    ARCH="arm64"
+    ;;
+  armv7l|arm)
+    ARCH="arm"
+    ;;
+  *)
+    echo "Not supported architecture. Exiting..."
+    exit 1
+esac
+
+VERSION_NUMBER=${VERSION:1}
+TF_PLUGIN_DIR="${HOME}/.terraform.d/plugins/clumio.com/providers/clumio/${VERSION_NUMBER}/${OS}_${ARCH}"
+if ! mkdir -p ${TF_PLUGIN_DIR}; then
+   echo "Error creating directory ${TF_PLUGIN_DIR}. Exiting..."
+  exit 1
+fi
+
+PROVIDER_NAME=terraform-provider-clumio_${VERSION_NUMBER}_${OS}_${ARCH}
+BINARY="https://github.com/clumio-code/terraform-provider-clumio/releases/download/${VERSION}/${PROVIDER_NAME}.zip"
+
+if ! wget ${BINARY}; then
```

Comment:
FYI wget is not standard on macOS, the OS ships with curl and we install wget through brew for convince on the Clumio Engineering laptops. We should ensure that we can download with either wget or curl present.